ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,10 @@
 Please answer the following questions before submitting your issue. Thanks!
 
-### What version of the DCOS CLI are you using (`dcos --version`)?
+### What version of DCOS + DCOS CLI are you using (`dcos --version`)?
+If DCOS CLI version < 0.4.5, DCOS version can be found at `http://dcos_url/dcos-metada/dcos-version.json`
 
 
-### What version of DCOS are you using?
+### What version of DCOS are you using (dcos_url/dcos-metada)?
```

Comment:
We should just remove this since you have instructions for getting the DC/OS version if they have an old CLI.